### ISSUE TITLE
Menu (small screen) and Cart side drawer content not accessible by keyboard.

### ISSUE THREAD

**Issue by g1st** [chingu-voyages/v8-bears-team-14#24]
User should be able to tab through menu items or links in cart drawer.

**Comment by nikrb** [comment]
re tabbing, I think the work around I used when linting was to add the tabIndex attrib.
